ConfigurationSystem/Client/ConfigurationServerClient.py

```diff
@@ -0,0 +1,13 @@
+"""
+  Custom Client for Configuration System
+"""
+
+from DIRAC.Core.Base.Client import Client
+
+
+class ConfigurationServerClient(Client):
+  """
+    Placeholder client to speak with ConfigurationServer.
+  """
+
+  pass
```

Comment:
Using here
```
self.setServer('Configuration/Server')
```
wouldn't work?

Reply:
Sometime component need to provide an explicit url (for example, the master configuration server choose a slave server to ping it). This line may override the url given from `**kwargs` with a random url from CS.
But I can add this:
```
  def __init__(self, **kwargs):
    Client.__init__(self, **kwargs)
    if not self.getServer():
      self.setServer('Configuration/Server')
```
So by default it use an URL from the CS, but it still possible to give another URL on the keywords arguments.